spec.html

```diff
@@ -23866,11 +23866,14 @@ <h1>ECMAScript Standard Built-in Objects</h1>
 <!-- es6num="18" -->
 <emu-clause id="sec-global-object">
   <h1>The Global Object</h1>
-  <p>The unique <em>global object</em> is created before control enters any execution context.</p>
-  <p>The global object does not have a [[Construct]] internal method; it is not possible to use the global object as a constructor with the `new` operator.</p>
-  <p>The global object does not have a [[Call]] internal method; it is not possible to invoke the global object as a function.</p>
-  <p>The value of the [[Prototype]] internal slot of the global object is implementation-dependent.</p>
-  <p>In addition to the properties defined in this specification the global object may have additional host defined properties. This may include a property whose value is the global object itself; for example, in the HTML document object model the `window` property of the global object is the global object itself.</p>
+  <p>The global object:</p>
+  <ul>
+    <li>is created before control enters any execution context.</li>
```

Comment:
This removes the word "unique", is that intended?

Reply:
Yes, intended. To say "unique" for this object but not the others suggests a distinction that I don't think is there, e.g. that there's a single global object shared by all realms.